docusaurus/docs/dev-docs/deployment/heroku.md

```diff
@@ -102,14 +102,22 @@ export default ({ env }) => ({
 
 ```js
 // Path: ./config/env/production/server.js
+// starting from Strapi v 4.6.1 server.js has to be the following
 
 module.exports = ({ env }) => ({
-        proxy: true,
-        url: env('MY_HEROKU_URL'), // Sets the public URL of the application.
-        app: { 
-          keys: env.array('APP_KEYS')
-        },
-    });
+  proxy: true,
+  host: "0.0.0.0",
```

Comment:
I guess we could also set this as a `env` not that it needs to change.
But it would make it more dynamic.
We are also using `process.env.PORT` where we could make it more standard to say `env("PORT")` IMO

Reply:
Thanks, problem is that the environment variables are set in the HEROKU settings dashboard, the current solution doesnt work when deploying the app in Heroku but it givves the following error: 
Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
while the proposed change it works at least on Strapi version > 4.6.0